src/components/elementView/elementTab.tsx

```diff
@@ -295,6 +296,22 @@ class ElementTab extends React.Component<ElementTabProps, ElementViewState> {
       console.log(this.state.isOpen);
     });
   };
+
+  private createChangePlan = (
```

Comment:
also I'm guessing this is named dynamically but it says "add change-plan" and "create change-plan" and it's not hyphenated anywhere else. do we care?

Reply:
i think front end method names consistently use create, but then the backend endpoints use add, hence the discrepancy. im not mad about it

Reply:
sorry i meant on the front end in the actual form, like the button itself says "+ Add change-plan" and the form title is "create change-plan" and everywhere else on the front end its written as "Change Plan"

Reply:
ohhhhh yes i am tracking that  - that is because i dynamically load it from the path name and routes can't have spaces. i will make a task for this as a lower priority refactor.